pkg/monitoring/rules/alerts/health_alerts.go

```diff
@@ -15,7 +15,7 @@ func healthAlerts() []promv1.Rule {
 			Alert: "OperatorConditionsUnhealthy",
 			Expr:  intstr.FromString(fmt.Sprintf("kubevirt_hco_system_health_status == %f", metrics.SystemHealthStatusError)),
 			Annotations: map[string]string{
-				"description": "HCO and its secondary resources are in a critical state due to {{ $labels.reason }}.",
+				"description": "HCO and its secondary resources are in a critical state due to system error.",
```

Comment:
You can add here and below:
Go to Installed Operators -> kubevirt-hyperconverged-operator -> HyperConverged details and inspect the Conditions.

Reply:
+1 added